src/MailpitCommands.ts

```diff
@@ -296,6 +328,34 @@ class MailpitCommands {
 	): Cypress.Chainable<string> {
 		return this.setReadStatus(messages, false);
 	}
+
+	private waitForCondition<T>(
+		fn: () => Cypress.Chainable<T>,
+		condition: (result: T) => boolean,
+		options: { timeout?: number; interval?: number } = {},
+	): Cypress.Chainable<T> {
+		const timeout = options.timeout ?? Cypress.config("defaultCommandTimeout");
+		const interval = options.interval ?? 500;
```

Comment:
README.md says 1000

Reply:
Nope it's way to customise so readme shows if you need to customise then how you should do it. I'll update it to 500 for better experience